### ISSUE TITLE
[Bug][Plugin] Tons of exceptions thrown in latest version of the plugin

### ISSUE THREAD

**Issue by CC007** [manifold-systems/manifold#493]
**Describe the bug**
Manifold plugin causes tons of plugin exceptions during use and fails to highlight extension methods properly (see stacktrace below as example).

**To Reproduce**
1. install manifold plugin 2023.1.19
2. open https://github.com/CC007/CFGDemo as an intellij project

**Expected behavior**
No errors and working syntax highlighting

**Screenshots**
![afbeelding](https://github.com/manifold-systems/manifold/assets/5381337/e1be3e6f-8deb-4db4-8fa7-04654e4e3269)
![afbeelding](https://github.com/manifold-systems/manifold/assets/5381337/ca2d2b82-7484-40cb-aae2-7b9b357e53e1)

**Desktop (please complete the following information):**
- OS Type & Version: Windows 10
- Java/JDK version: openjdk-20.0.2
- IDE version (IntelliJ IDEA or Android Studio): IntelliJ 2023.2.1
- Manifold version: 2023.1.24
- Manifold IntelliJ plugin version: 2023.1.19

**Additional context**
The errors started after I updated the plugin from 2023.1.15 to 2023.1.19.
The code does compile and run, as seen at the screenshot at the bottom.

**Stack trace**
```
  provider: manifold.ij.extensions.ManAugmentProvider@51cbe4cc
  
  java.lang.NumberFormatException: Cannot parse null string
at java.base/java.lang.Integer.parseInt(Integer.java:630)
at java.base/java.lang.Integer.valueOf(Integer.java:999)
at manifold.ij.util.ComputeUtil.computePrimitiveValue(ComputeUtil.java:89)
at manifold.ij.util.ComputeUtil.computeLiteralValue(ComputeUtil.java:53)
at manifold.ij.extensions.StubBuilder.addAnnotations(StubBuilder.java:412)
at manifold.ij.extensions.StubBuilder.addAnnotations(StubBuilder.java:398)
at manifold.ij.extensions.StubBuilder.addMethod(StubBuilder.java:281)
at manifold.ij.extensions.StubBuilder.makeSrcClass(StubBuilder.java:75)
at manifold.ij.extensions.StubBuilder.make(StubBuilder.java:61)
at manifold.ij.extensions.ManAugmentProvider.addMethods(ManAugmentProvider.java:279)
at manifold.ij.extensions.ManAugmentProvider.addMethods(ManAugmentProvider.java:220)
at manifold.ij.extensions.ManAugmentProvider.addMethods(ManAugmentProvider.java:182)
at manifold.ij.extensions.ManAugmentProvider.lambda$_getAugments$1(ManAugmentProvider.java:156)
at com.intellij.psi.util.CachedValuesManager$1.compute(CachedValuesManager.java:158)
at com.intellij.psi.impl.PsiCachedValueImpl.doCompute(PsiCachedValueImpl.java:37)
at com.intellij.util.CachedValueBase.lambda$getValueWithLock$3(CachedValueBase.java:240)
at com.intellij.util.CachedValueBase.computeData(CachedValueBase.java:43)
at com.intellij.util.CachedValueBase.lambda$getValueWithLock$4(CachedValueBase.java:240)
at com.intellij.openapi.util.RecursionManager$1.computePreventingRecursion(RecursionManager.java:110)
at com.intellij.openapi.util.RecursionGuard.doPreventingRecursion(RecursionGuard.java:27)
at com.intellij.openapi.util.RecursionManager.doPreventingRecursion(RecursionManager.java:65)
at com.intellij.util.CachedValueBase.getValueWithLock(CachedValueBase.java:241)
at com.intellij.psi.impl.PsiCachedValueImpl.getValue(PsiCachedValueImpl.java:27)
at com.intellij.util.CachedValuesManagerImpl.getCachedValue(CachedValuesManagerImpl.java:69)
at com.intellij.psi.util.CachedValuesManager.getCachedValue(CachedValuesManager.java:155)
at manifold.ij.extensions.ManAugmentProvider._getAugments(ManAugmentProvider.java:142)
at manifold.ij.extensions.ManAugmentProvider.lambda$getAugments$0(ManAugmentProvider.java:102)
at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:909)
at manifold.ij.extensions.ManAugmentProvider.getAugments(ManAugmentProvider.java:102)
at com.intellij.psi.augment.PsiAugmentProvider.lambda$collectAugments$3(PsiAugmentProvider.java:134)
at com.intellij.psi.augment.PsiAugmentProvider.forEach(PsiAugmentProvider.java:257)
at com.intellij.psi.augment.PsiAugmentProvider.collectAugments(PsiAugmentProvider.java:133)
at com.intellij.psi.impl.source.ClassInnerStuffCache.calcMethods(ClassInnerStuffCache.java:143)
at com.intellij.psi.impl.source.ClassInnerStuffCache.lambda$getMethods$1(ClassInnerStuffCache.java:51)
at com.intellij.psi.util.CachedValuesManager.lambda$getProjectPsiDependentCache$0(CachedValuesManager.java:134)
at com.intellij.psi.util.CachedValuesManager$1.compute(CachedValuesManager.java:158)
at com.intellij.psi.impl.PsiCachedValueImpl.doCompute(PsiCachedValueImpl.java:37)
at com.intellij.util.CachedValueBase.lambda$getValueWithLock$3(CachedValueBase.java:240)
at com.intellij.util.CachedValueBase.computeData(CachedValueBase.java:43)
at com.intellij.util.CachedValueBase.lambda$getValueWithLock$4(CachedValueBase.java:240)
at com.intellij.openapi.util.RecursionManager$1.computePreventingRecursion(RecursionManager.java:110)
at com.intellij.openapi.util.RecursionGuard.doPreventingRecursion(RecursionGuard.java:27)
at com.intellij.openapi.util.RecursionManager.doPreventingRecursion(RecursionManager.java:65)
at com.intellij.util.CachedValueBase.getValueWithLock(CachedValueBase.java:241)
at com.intellij.psi.impl.PsiCachedValueImpl.getValue(PsiCachedValueImpl.java:27)
at com.intellij.util.CachedValuesManagerImpl.getCachedValue(CachedValuesManagerImpl.java:69)
at com.intellij.psi.util.CachedValuesManager.getCachedValue(CachedValuesManager.java:155)
at com.intellij.psi.util.CachedValuesManager.getProjectPsiDependentCache(CachedValuesManager.java:132)
at com.intellij.psi.impl.source.ClassInnerStuffCache.getMethods(ClassInnerStuffCache.java:51)
at com.intellij.psi.impl.compiled.ClsClassImpl.getMethods(ClsClassImpl.java:173)
at com.intellij.psi.impl.PsiClassImplUtil$MemberType.getMembers(PsiClassImplUtil.java:174)
at one.util.streamex.AbstractStreamEx.lambda$flatArray$17(AbstractStreamEx.java:683)
at java.base/java.util.stream.ReferencePipeline$7$1.accept(ReferencePipeline.java:273)
at java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1625)
at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509)
at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:575)
at java.base/java.util.stream.AbstractPipeline.evaluateToArrayNode(AbstractPipeline.java:260)
at java.base/java.util.stream.ReferencePipeline.toArray(ReferencePipeline.java:616)
at one.util.streamex.StreamEx.toArray(StreamEx.java:871)
at com.intellij.psi.impl.PsiClassImplUtil$MemberCache.lambda$new$3(PsiClassImplUtil.java:352)
at com.intellij.util.containers.ConcurrentFactoryMap$2.create(ConcurrentFactoryMap.java:174)
at com.intellij.util.containers.ConcurrentFactoryMap.get(ConcurrentFactoryMap.java:40)
at com.intellij.psi.impl.PsiClassImplUtil$MemberCache.getAllMembers(PsiClassImplUtil.java:386)
at com.intellij.psi.impl.PsiClassImplUtil.getAllByMap(PsiClassImplUtil.java:166)
at com.intellij.psi.impl.PsiClassImplUtil.getAllMethods(PsiClassImplUtil.java:55)
at com.intellij.psi.impl.compiled.ClsClassImpl.getAllMethods(ClsClassImpl.java:250)
at manifold.ij.extensions.ManJavaResolveCache.getBinaryType(ManJavaResolveCache.java:241)
at manifold.ij.extensions.ManJavaResolveCache.getTypeForOverloadedBinaryOperator(ManJavaResolveCache.java:189)
at manifold.ij.extensions.ManJavaResolveCache.getTypeDirectly(ManJavaResolveCache.java:101)
at manifold.ij.extensions.ManJavaResolveCache.lambda$getType$1(ManJavaResolveCache.java:93)
at com.intellij.psi.util.CachedValuesManager.lambda$getProjectPsiDependentCache$0(CachedValuesManager.java:134)
at com.intellij.psi.util.CachedValuesManager$1.compute(CachedValuesManager.java:158)
at com.intellij.psi.impl.PsiCachedValueImpl.doCompute(PsiCachedValueImpl.java:37)
at com.intellij.util.CachedValueBase.lambda$getValueWithLock$3(CachedValueBase.java:240)
at com.intellij.util.CachedValueBase.computeData(CachedValueBase.java:43)
at com.intellij.util.CachedValueBase.lambda$getValueWithLock$4(CachedValueBase.java:240)
at com.intellij.openapi.util.RecursionManager$1.computePreventingRecursion(RecursionManager.java:110)
at com.intellij.openapi.util.RecursionGuard.doPreventingRecursion(RecursionGuard.java:27)
at com.intellij.openapi.util.RecursionManager.doPreventingRecursion(RecursionManager.java:65)
at com.intellij.util.CachedValueBase.getValueWithLock(CachedValueBase.java:241)
at com.intellij.psi.impl.PsiCachedValueImpl.getValue(PsiCachedValueImpl.java:27)
at com.intellij.util.CachedValuesManagerImpl.getCachedValue(CachedValuesManagerImpl.java:69)
at com.intellij.psi.util.CachedValuesManager.getCachedValue(CachedValuesManager.java:155)
at com.intellij.psi.util.CachedValuesManager.getProjectPsiDependentCache(CachedValuesManager.java:132)
at manifold.ij.extensions.ManJavaResolveCache.getType(ManJavaResolveCache.java:93)
at com.intellij.psi.impl.source.tree.java.PsiBinaryExpressionImpl.getType(PsiBinaryExpressionImpl.java:90)
at com.siyeh.ig.performance.StringConcatenationInLoopsInspection$StringConcatenationInLoopsVisitor.checkExpression(StringConcatenationInLoopsInspection.java:145)
at com.siyeh.ig.performance.StringConcatenationInLoopsInspection$StringConcatenationInLoopsVisitor.visitPolyadicExpression(StringConcatenationInLoopsInspection.java:116)
at com.intellij.psi.JavaElementVisitor.visitBinaryExpression(JavaElementVisitor.java:45)
at com.intellij.psi.impl.source.tree.java.PsiBinaryExpressionImpl.accept(PsiBinaryExpressionImpl.java:134)
at com.intellij.codeInsight.daemon.impl.InspectionRunner.lambda$processInOrder$10(InspectionRunner.java:334)
at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1133)
at com.intellij.codeInsight.daemon.impl.InspectionRunner.lambda$processInOrder$11(InspectionRunner.java:342)
at com.intellij.openapi.application.impl.ReadMostlyRWLock.executeByImpatientReader(ReadMostlyRWLock.java:201)
at com.intellij.openapi.application.impl.ApplicationImpl.executeByImpatientReader(ApplicationImpl.java:207)
at com.intellij.codeInsight.daemon.impl.InspectionRunner.lambda$processInOrder$12(InspectionRunner.java:348)
at com.intellij.util.AstLoadingFilter.forceAllowTreeLoading(AstLoadingFilter.java:159)
at com.intellij.util.AstLoadingFilter.forceAllowTreeLoading(AstLoadingFilter.java:151)
at com.intellij.codeInsight.daemon.impl.InspectionRunner.lambda$processInOrder$13(InspectionRunner.java:311)
at com.intellij.util.AstLoadingFilter.disallowTreeLoading(AstLoadingFilter.java:130)
at com.intellij.util.AstLoadingFilter.disallowTreeLoading(AstLoadingFilter.java:119)
at com.intellij.codeInsight.daemon.impl.InspectionRunner.lambda$processInOrder$14(InspectionRunner.java:311)
at com.intellij.concurrency.JobLauncherImpl$1MyProcessQueueTask.lambda$call$0(JobLauncherImpl.java:314)
at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$executeProcessUnderProgress$13(CoreProgressManager.java:604)
at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:679)
at com.intellij.openapi.progress.impl.CoreProgressManager.computeUnderProgress(CoreProgressManager.java:635)
at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:603)
at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:61)
at com.intellij.concurrency.JobLauncherImpl$1MyProcessQueueTask.call(JobLauncherImpl.java:299)
at com.intellij.concurrency.JobLauncherImpl$1MyProcessQueueTask.call(JobLauncherImpl.java:285)
at java.base/java.util.concurrent.ForkJoinTask$AdaptedCallable.exec(ForkJoinTask.java:1428)
at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)
at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)
at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)
at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)
```
The above stacktrace is the most common (sometimes with slightly different `at ...`, but with the same error. Some of them also refer to `provider: manifold.ij.extensions.ManPropertiesAugmentProvider@677b7f39` instead

**Comment by rsmckinney** [comment]
Hi @CC007. I think I know what change may have caused a regression here, but the project you link to ([https://github.com/CC007/CFGDemo](https://github.com/CC007/CFGDemo)) doesn't appear to exist. I'd like to make sure my change fixes the issue you have.

**Comment by rsmckinney** [comment]
Turns out this is already fixed and ready for the next IJ plugin release, coming tonight :)

**Comment by CC007** [comment]
> Hi @CC007. I think I know what change may have caused a regression here, but the project you link to (https://github.com/CC007/CFGDemo) doesn't appear to exist. I'd like to make sure my change fixes the issue you have.

Ah, I didn't realize that I made the repo private. I'll change that.

**Comment by CC007** [comment]
Should be public now

**Comment by rsmckinney** [comment]
Fix available in latest IJ plugin releases 2023.1.25, 2022.3.42, 2022.2.42.  These are pending JetBrains approval, which usually takes less than one business day.

**Comment by CC007** [comment]
Fix confirmed.
